contrib/dimfeld/httptreemux/v5/option.go

```diff
@@ -0,0 +1,85 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+// Package httptreemux provides functions to trace the dimfeld/httptreemux/v5 package (https://github.com/dimfeld/httptreemux).
+package httptreemux
+
+import (
+	"math"
+	"net/http"
+
+	"github.com/dimfeld/httptreemux/v5"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/globalconfig"
+)
+
+type routerConfig struct {
+	serviceName   string
+	spanOpts      []ddtrace.StartSpanOption
+	analyticsRate float64
```

Comment:
We actually don't need this anymore, as this is a deprecated feature. No need to remove it from other library integrations right now, but also no need to introduce this in new integrations. So you can just remove this variable and all usages of it in this package.

Reply:
- [ ] remove `analyticsRate` and usages